### ISSUE TITLE
Maybe add bootstraptoggle plugin?

### ISSUE THREAD

**Issue by bborgesr** [rstudio/shiny#1417]
http://www.bootstraptoggle.com/

**Comment by daattali** [comment]
I agree, I've also noticed that toggles are a very common feature. I was actually looking last week at this exact library for shiny.  :)
